docs/source/tutorials/python_api_demo.ipynb

```diff
@@ -22,8 +22,7 @@
     {
      "name": "stderr",
      "output_type": "stream",
-     "text": [
```

Comment:
Why are any edits to this file in this PR?

Reply:
just autoformat, will remove